### DISCUSSION TITLE
Furo does not wrap type hints correctly in documentation

### DISCUSSION THREAD

**Discussion by GoogolGenius** [pradyunsg/furo#454] [Potential Issue] [answered]
**Page overflow bug**

There is a bug where in this case, having extensive type hinting information causes the page to overflow. Attached are screenshots which shows a horizontal overflow bar as well as the type hints being cut in documentation. 

This was found to happen on iPadOS 15.5 Safari. Specific device information including an iPad Pro 11 from 2018. Additionally, iPadOS by default renders web pages in desktop mode in Safari. 

[This](https://hikari-py--1185.org.readthedocs.build/en/1185/reference/hikari/messages/index.html?highlight=messages#module-hikari.messages) is the documentation page where the issue occurred.

![ED6E9060-8EE7-4893-96CA-FE737D75B6C6](https://user-images.githubusercontent.com/81032623/173236715-feec5cb3-2045-4bd6-9be1-f3d19cd27a38.jpeg)
![3EF952F1-ABAC-4698-88DB-D1D90A1211A0](https://user-images.githubusercontent.com/81032623/173236713-837374d6-a0d2-473c-9215-acbe3f814c7a.jpeg)

**Comment by pradyunsg** [comment, marked as answer]
Looking again at https://hikari-py--1185.org.readthedocs.build/en/1185/reference/hikari/messages/index.html#hikari.messages.PartialMessage.edit, this got fixed at some point. Thanks for flagging this @GoogleGenius! ^>^
